gradle/project.gradle.kts

```diff
@@ -183,11 +197,11 @@ tasks {
                         }
                     }
                     if (!kotlinc.exists())
-                        throw RuntimeException("Kotlinc has not been downloaded successfully! Maybe the remote file is not a zip?")
+                        throw IOException("Kotlinc has not been downloaded successfully! Maybe the remote file is not a zip?")
 
                     File("$workingDir/kotlinc/bin/kotlinc").setExecutable(true) // Allow kotlinc to be executed on Unix
                 } catch (e: FileNotFoundException) {
-                    throw RuntimeException("The remote file could not be found", e)
+                    throw IOException("The remote file could not be found", e)
```

Comment:
but if the problem is different, then it should throw a different exception. I'm not merely rethrowing the exception since I want to have a custom message. Without the `--stacktrace` flag only the message of the error will be visible

Reply:
and, actually, it only throws IOException now